indexer/httpd/src/graphql.rs

```diff
@@ -22,7 +22,6 @@ pub mod mutation;
 pub mod query;
 pub mod subscription;
 pub mod telemetry;
-pub mod types;
```

Comment:
@penso Files in this module are unused and are duplicates of `grug_httpd::graphql::types`. Therefore I deleted them.